v2/netconf/server/ssh/trace_test.go

```diff
@@ -0,0 +1,13 @@
+package ssh
+
+import (
+	"errors"
+	"testing"
+)
+
+func TestDefaultHooksForUntestableExceptions(t *testing.T) {
+
+	hooks := DefaultLoggingHooks
```

Comment:
no assertions?

Reply:
Nothing to assert.   
As above, this test is only there to eke out a bit more coverage - the hooks under test cannot be exercised during normal testing, hence the name of the test - TestDefaultHooksForUntestableExceptions.